### ISSUE TITLE
AWS access for Bryan Alexander

### ISSUE THREAD

**Issue by balexandr** [department-of-veterans-affairs/va.gov-team#74567]
### COR Name

Eunice Garcia

### Vendor Onboarding Representative Name

Other - please specify in 'Additional Notes'

### Your Name

Bryan Alexander

### Your Email

bryan.alexander@adhocteam.us

### Team, Role, and Company of the target individual

IVC Forms (CHAMPVA), Software Engineer, Ad Hoc

### Product Manager (PM) name and email

Premal Shah

### Product Owner (PO) name and email

Premal Shah

### Desired AWS Access

The IVC Form team will be pushing the digitized forms to a S3 service. Bucket TBD but we all need accounts in AWS before we can be granted access to any bucket

### Access Expiration

ongoing

### E-QIP Transmittal Confirmation

<img width="1440" alt="Screenshot 2024-01-31 at 3 56 39 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/5070341/eb9f5d51-f293-4587-8cc6-f6854023abca">

### Additional Notes

I'm not entirely sure who my "Vendor Onboarding Representative Name" possibly Devi Hill with Agile Six? Or Jill Langan with Ad Hoc.

### User must exist in a roster before AWS access can be granted. See [AWS access documentation](https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools#AWS-Access-Request) for the steps involved.

- [x] Search for VFS team member in [Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] Or search for user on the [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)
- [ ] If user is on a VFS team but not a team member in Atlas, add the 'NOT YET' label and instruct them to start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] If a user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added.
- [x] Comment in this issue saying which roster the user is listed in.

**Comment by barbarello** [comment]
user is neither in Atlas nor in VFS roster

**Comment by barbarello** [comment]
@balexandr can not find you in Atlas as of yet

**Comment by balexandr** [comment]
@barbarello I am in Atlas now FYI. Let me know if I need to create a new ticket for S3 access after my IAM user is created in AWS.

**Comment by barbarello** [comment]
user in Atlas

**Comment by balexandr** [comment]
![Screenshot 2024-01-31 at 3 53 47 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1422922/6d7e4d35-937b-4a44-b7f5-cecfd17eb811)

**Comment by barbarello** [comment]
e-qip transmittal notice is valid
